docs/sample_config.yaml

```diff
@@ -2729,3 +2729,36 @@ redis:
   # Optional password if configured on the Redis instance
   #
   #password: <secret_password>
+
+
+
+
+## Background Updates ##
+
+# Background updates are database updates that are run in the background in batches.
+# The duration, minimum batch size, default batch size, whether to sleep between batches and if so, how long to
+# sleep can all be configured. This is helpful to speed up or slow down the updates.
+
+# How long in milliseconds to run a batch of background updates for. Defaults to 100. Uncomment and set
+# a time to change the default.
+#
+#background_update_duration_ms: 500
+
+# Whether to sleep between updates. Defaults to True. Uncomment to change the default.
+#
+#sleep_enabled: false
+
+# If sleeping between updates, how long in milliseconds to sleep for. Defaults to 1000. Uncomment
+# and set a duration to change the default.
+#
+#sleep_duration_ms: 300
+
+# Minimum size a batch of background updates can be. Must be greater than 0. Defaults to 1. Uncomment and
+# set a size to change the default.
+#
+#min_batch_size: 10
+
+# The batch size to use for the first iteration of a new background update. The default is 100.
+# Uncomment and set a size to change the default.
+#
+#default_batch_size: 50
```

Comment:
I think this would probably be best grouped under a key? e.g.;

```yaml
background_updates:
    min_batch_size: 10
    default_batch_size: 50
```

Reply:
Great suggestion, I have implemented it.